### ISSUE TITLE
Improve docs and tests for "tiled" and FMRC aggregations

### ISSUE THREAD

**Issue by cwardgar** [Unidata/thredds#785]
From #778.

`tiled` aggregations are only mentioned **very** briefly in [two](https://github.com/Unidata/thredds/blob/5.0.0/docs/website/netcdf-java/ncml/AnnotatedSchema4.adoc#netcdf-element) [places](https://github.com/Unidata/thredds/blob/5.0.0/docs/website/netcdf-java/ncml/AnnotatedSchema4.adoc#aggregationtype-enumeration) in `AnnotatedSchema4.adoc`. They're also mentioned in the [NcML reference document](https://github.com/Unidata/thredds/blob/5.0.0/docs/website/netcdf-java/ncml/reference.adoc), but that thing's obviously an unfinished stub (we should probably nuke it). There are no examples for them in the docs, but there is a [test](https://github.com/Unidata/thredds/blob/5.0.0/cdm/src/test/java/ucar/nc2/ncml/TestAggTiled.java).

FMRC aggregations are listed in 6 separate reference docs: [Aggregation.adoc](https://github.com/Unidata/thredds/blob/5.0.0/docs/website/netcdf-java/ncml/Aggregation.adoc), [AnnotatedSchema4.adoc](https://github.com/Unidata/thredds/blob/5.0.0/docs/website/netcdf-java/ncml/AnnotatedSchema4.adoc), [Formica.adoc](https://github.com/Unidata/thredds/blob/5.0.0/docs/website/netcdf-java/ncml/Formica.adoc), the [NcML reference document](https://github.com/Unidata/thredds/blob/5.0.0/docs/website/netcdf-java/ncml/reference.adoc), [Fmrc.adoc](https://github.com/Unidata/thredds/blob/5.0.0/docs/website/netcdf-java/reference/FeatureDatasets/Fmrc.adoc), and [FmrcCollection.adoc](https://github.com/Unidata/thredds/blob/5.0.0/docs/website/tds/reference/collections/FmrcCollection.adoc). Unfortunately, there's a lot of cruft and duplication in there. For example, the aggregation type `forecastModelRunSingleCollection` is hardly mentioned at all. Why?

**Comment by cofinoa** [comment]
yes, my experience with TiledAggregation started when I discovered it reading the code for the aggregation. That could be the reason, is scarced mentioned in the doc, I was asking about it in th email list. I'm still have to make many try & error and "readthecode" everytme I'm using it. 

The FMRC collection agregations are time oriented ones (forecastModelRuntime and/or forecastTime) but tiled aggregation is the general case for them and for outer dimension aggregation. It worth it to mantain the test and improve the doc with more examples.
